src/Eccube/Resource/template/default/Shopping/index.twig

```diff
@@ -58,8 +57,12 @@ $(function() {
         var name = [];
         var input = [];
         var customerin = [];
+        var customerEditButton = $('#customer');
```

Comment:
when you use "jquery variable", you should put `$`.

`customerEditButton`  -> `$customerEditButton`